pkg/controller/rbac_jobs.go

```diff
@@ -73,6 +74,12 @@ func (c *StashController) ensureCronJobClusterRole() error {
 				Resources: []string{api_v1beta1.ResourcePluralBackupConfiguration},
 				Verbs:     []string{"*"},
 			},
+			{
```

Comment:
https://github.com/kmodules/security-policy-api/issues/3

For Openshift, we need to add s/a name to the SCC.users list. It will be equivalent to 

```
oc adm policy add-scc-to-user <scc_name> -z <sa_name>
```